JustTweak/Classes/Core/EphemeralConfiguration.swift

```diff
@@ -5,6 +5,7 @@
 
 import Foundation
 
+@objcMembers
 @objc(JEEphemeralConfiguration) public class EphemeralConfiguration: NSObject, MutableTweaksConfiguration {
```

Comment:
Do we need @objc if using @objcmembers?

Reply:
I would remove all the things JE-prefixed, as done for other repositories. Let's live in the future.

Reply:
I add both so I don't break the client using it. If we decide to remove, happy to do.

Reply:
I would propose to not change the interface, leave it